datafusion/sqllogictest/test_files/parquet.slt

```diff
@@ -263,13 +263,13 @@ LIMIT 10;
 0 NULL Timestamp(Millisecond, Some("UTC"))
 0 2014-08-27T14:00:00Z Timestamp(Millisecond, Some("UTC"))
 0 2014-08-27T14:00:00Z Timestamp(Millisecond, Some("UTC"))
-4 2014-08-27T14:00:00Z Timestamp(Millisecond, Some("UTC"))
-0 2014-08-27T14:00:00Z Timestamp(Millisecond, Some("UTC"))
 0 2014-08-27T14:00:00Z Timestamp(Millisecond, Some("UTC"))
+12 2014-08-27T14:00:00Z Timestamp(Millisecond, Some("UTC"))
+2 2014-08-27T14:00:00Z Timestamp(Millisecond, Some("UTC"))
 0 2014-08-27T14:00:00Z Timestamp(Millisecond, Some("UTC"))
 14 2014-08-27T14:00:00Z Timestamp(Millisecond, Some("UTC"))
-0 2014-08-27T14:00:00Z Timestamp(Millisecond, Some("UTC"))
-0 2014-08-27T14:00:00Z Timestamp(Millisecond, Some("UTC"))
+5 2014-08-27T14:00:00Z Timestamp(Millisecond, Some("UTC"))
+1 2014-08-27T14:00:00Z Timestamp(Millisecond, Some("UTC"))
```

Comment:
So it's not just the ordering. The values in the first `count` column is incorrect.